Formula/mycli.rb

```diff
@@ -20,6 +20,8 @@ class Mycli < Formula
   end
 
   depends_on "rust" => :build
+  # Click does not support bash version older than 4.4
+  depends_on "bash"
```

Comment:
maybe we can conditionally generate bash completion depending on system bash version.